images/macos/provision/core/android-toolsets.sh

```diff
@@ -17,14 +17,24 @@ function filter_components_by_version {
     done
 }
 
+function get_latest_major_ndk {
+    majorVersion=$1
+
+    ndkVersions=($(${SDKMANAGER} --list | grep "ndk;${majorVersion}.*" | cut -d"|" -f 1 | sort -V | cut -d";" -f 2))
+    ndkVersion="${ndkVersions[@]:(-1)}"
+
```

Comment:
Suggest changing it to one liner
`ndkVersion=$(${SDKMANAGER} --list | grep "ndk;${majorVersion}.*" | awk '{gsub("ndk;", ""); print $1}' | sort -V | tail -n1)`

Reply:
Yes, sure. Thank you.